cs_api_io/cs_api_io.ml

```diff
@@ -1,79 +1,53 @@
-let get_ctx ~verify =
-  let open Lwt.Infix in
-  Conduit_lwt_unix.init ~verify () >|= fun ctx ->
-  Cohttp_lwt_unix.Client.custom_ctx ~ctx ()
+module Response = struct
+  type t =
+    { code : int
+    ; response : string ref }
```

Comment:
- As far as I can tell, there's nothing in the code that requires this to be a `ref`.  If that's indeed the case, what about making it a simple string?
- What about renaming this record field to `body`?

Reply:
Yes, good catch, that is a left-over of an older version. Changed it to match your suggestions.